src/message.js

```diff
@@ -215,9 +215,6 @@ export class Message {
           // do not check key expiration to allow decryption of old messages
           const decryptionKeyPackets = (await decryptionKey.getDecryptionKeys(pkeskPacket.publicKeyID, null, undefined, config)).map(key => key.keyPacket);
           await Promise.all(decryptionKeyPackets.map(async function(decryptionKeyPacket) {
-            if (!decryptionKeyPacket || decryptionKeyPacket.isDummy()) {
-              return;
-            }
```

Comment:
...by throwing an exception here instead, and then expanding the `try/catch`es below to cover this (and the one below as well, since theoretically speaking you could have one key packet that's decrypted and one that's not).